packages/oui-form-actions/src/form-actions.controller.js

```diff
@@ -0,0 +1,29 @@
+export default class {
+    constructor ($attrs, ouiFormActions) {
+        "ngInject";
+
+        this.$attrs = $attrs;
+        this.config = ouiFormActions;
+    }
+
+    $onInit () {
+        // Support presence of attribute 'disabled'
+        if (angular.isDefined(this.$attrs.disabled) && angular.isUndefined(this.disabled)) {
```

Comment:
There is a helper for that.
Example: https://github.com/ovh-ux/ovh-ui-angular/blob/master/packages/oui-action-menu/src/action-menu.controller.js#L16

Reply:
Oh nice. Will do a PR to pass this into oui-button.

Reply:
👍